packages/components/src/List/List.component.js

```diff
@@ -118,7 +113,6 @@ function List({ displayMode, id, list, toolbar, defaultHeight, t, renderers, row
 					displayMode={displayMode}
 					defaultHeight={defaultHeight}
 					rowHeight={rowHeight}
-					t={t}
```

Comment:
You removed this one, but you forgot the one you pass to <ListToolbar> that is outdated.
Don't forget to remove the variable declaration.